package.json

```diff
@@ -3,7 +3,9 @@
     "version": "0.0.1",
     "private": true,
     "description": "Buidl1 contracts monorepo.",
-    "scripts": {},
+    "scripts": {
+        "build":"set -e;for i in investment-pool governance-pool distribution-pool;do yarn workspace @buidlone/$i build;done"
```

Comment:
Any reason why this can't be `yarn workspaces run build`?

Reply:
`yarn workspaces run build` will build every package in the workspace. I think it would be better to separate contracts and modules such as `subgraph`. I will make two different scripts:

1. `build` - building all the modules (including SC)
2. `build:contracts` - building only contracts in package dir

Reply:
Ok, makes sense